aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,320 @@
+
+#### Test the app
+
+1. Open the project in Visual Studio by double clicking on the `GitHubBrowser.sln` solution file in the project folder.
+1. Launch the app by clicking the run button at the top of Visual Studio. The project includes a Docker file, which means you can select the arrow next to the run button to start the app using either a Docker Desktop setup or the standard ASP.NET Core local web server.
+
+When the app launches in the browser, you can use the search form to browse for GitHub repositories by name.
+
+:::image type="content" source="./media/scaling-app-screenshot.png" alt-text="A screenshot showing the GitHub Explorer app.":::
+
+## Deploy the app to Azure Container Apps
+
+You'll use Visual Studio to deploy the app to Azure Container Apps. Container apps provide a managed service designed to simplify hosting containerized apps and microservices.
+
+1. Inside the Visual Studio solution explorer, right click on the top level project node and select **Publish**.
+1. In the publishing dialog, select **Azure** as the deployment target, and then choose **Next**.
+1. For the specific target, select **Azure Container Apps (Linux)**, and then choose **Next**.
+1. You'll need to create a new container app to deploy to. Select the green **+** icon to open a new dialog and enter the following values:
+
+    :::image type="content" source="./media/scaling-deploy-visual-studio-small.png" lightbox="./media/scaling-deploy-visual-studio.png" alt-text="A screenshot showing Visual Studio deployment.":::
+
+    * **Container app name**: Leave the default value or enter a name of your choosing.
+    * **Subscription name**: Select the subscription you'd like to deploy to.
+    * **Resource group**: Select **New** and create a new resource group called *msdocs-scalable-razor*.
+    * **Container apps environment**: Select **New** to open the container apps environment dialog and enter the following values:
+        * **Environment name**: Leave the default value or enter a name of your choosing.
+        * **Location**: Select a location near you.
+        * **Azure Log Analytics Workspace**: Select **New** to open the log analytics workspace dialog.
+            * **Name**: Leave the default value or enter a name of your choosing.
+            * **Location**: Select a location near you and then choose **Ok** to close the dialog.
+        * Select **Ok** to close the container apps environment dialog.
+    * Select **Create** to close the original container apps dialog and Visual Studio creates the container app resource in Azure.
+1. Once the resource is created, make sure it's selected in the list of container apps, and then choose **Next**.
+1. You'll need to create an Azure Container Registry to store the published image artifact for your app. Select the green **+** icon on the container registry screen. Leave the default values, and then select **Create**.
```

Comment:
Need a red box, 2 pixels wide, around Create in the image.

Reply:
Done